changelog/getLocationAbsolutePath.dd

```diff
@@ -0,0 +1,7 @@
+The trait `getLocation` has been modified to accept a parameter to ask for an absolute path.
+
+Let `symbol` be a symbol known to the compiler, and *file* be the file in which it's declared:
+
+`__traits(getLocation, symbol)[0]` is the relative path to *file* to as it has always been.
```

Comment:
Why not always return the absolute path instead?

Reply:
Please let's have the possibility of making reproducible builds easier. Embedding absolute paths on the binary makes it less reproducible in different environments and requires tricks like building on a mocked/wrapped filesystem in a vfs/shmfs.